includes/class-bitcoin-donation-shortcode-voting.php

```diff
@@ -0,0 +1,228 @@
+                <div class="bitcoin-voting-form-container">
+                    <h3><?php echo esc_html($title ? $title : 'Bitcoin Voting'); ?></h3>
+                    <p><?php echo esc_html($description ? $description : 'What would you like to see more of on our blog?'); ?></p>
+                    <h4>Poll starting in: <?php echo $time_until_start; ?></h4>
+                </div>
+            </div>
+        <?php
+        } elseif ($now > $end_timestamp) {
+        ?>
+            <div id="bitcoin-voting-form" class="bitcoin-voting-form  <?php echo esc_attr($theme_class); ?>">
+                <div class="bitcoin-voting-form-container">
+                    <h3><?php echo esc_html($title ? $title : 'Voting Poll'); ?></h3>
+                    <p><?php echo esc_html($description ? $description : ''); ?></p>
+                    <div id="poll-results" class="poll-results" data-end-date="<?php echo $end_date; ?>" data-poll-id="<?php echo $poll_id; ?>">
+                        <?php
+                        for ($i = 1; $i <= min(4, $num_options ? $num_options : 4); $i++):
+                            if (isset($options[$i])):
+                        ?>
+                                <div class="poll-result">
+                                    <span><?php echo esc_html($options[$i]); ?> (<span class="vote-count" data-option="<?php echo $i; ?>">0</span> votes)</span>
+                                    <div class="progress">
+                                        <div class="progress-bar" data-option="<?php echo $i; ?>"></div>
+                                        <span class="progress-percentage" data-option="<?php echo $i; ?>"></span>
+                                    </div>
+
+                                </div>
+                        <?php endif;
+                        endfor; ?>
+                        <div class="poll-total-votes">
+                            <div class="poll-total-wrapper">
+                                Total votes:
+                                <div id="total-votes">
+                                </div>
+                            </div>
+                            <div class="end-text"><?php echo '<div>Poll closed</div>'; ?></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        <?php
+        } else {
+            $time_until_end = human_time_diff($now, $end_timestamp);
+        ?>
+            <div id="bitcoin-voting-form" class="bitcoin-voting-form <?php echo esc_attr($theme_class); ?>"
+                data-poll-id="<?php echo esc_attr($poll_id); ?>"
+                data-poll-amount="<?php echo esc_attr($amount ?: '0'); ?>"
+                data-one-vote="<?php echo esc_attr($one_vote) ?>">
+
+                <div class="bitcoin-voting-form-container">
+                    <div class="title-container">
+                        <h3><?php echo esc_html($title ? $title : 'Bitcoin Voting'); ?></h3>
```

Comment:
This can be improved:

```suggestion
                        <h3><?php echo esc_html($title ?: 'Bitcoin Voting'); ?></h3>
```